runtime/src/bank.rs

```diff
@@ -2764,6 +2765,32 @@ impl Bank {
         }
     }
 
+    #[allow(dead_code)]
+    fn hash_rewards_into_partitions(
+        stake_rewards: &StakeRewards,
+        rewarded_epoch: Epoch,
+        rewards: u64,
+        num_partitions: usize,
+        thread_pool: &ThreadPool,
+    ) -> Vec<StakeRewards> {
+        let seed = rewarded_epoch ^ rewards;
```

Comment:
why?

Reply:
a simple way to make the seed from epoch and total rewards.

Reply:
yeah but it's another piece of information that a client needs to lookup before they can query an address's epoch rewards.

its also difficult to reason about collision rate. something something birthday problem. though, this point is moot when we add parent blockhash to the seed.

Reply:
ok. make sense to me.
updated! now we only use parent block hash to seed the partitioning.